src/components/widgets/games.tsx

```diff
@@ -0,0 +1,90 @@
+import { Box } from "@mui/material"
+import { teteGray, teteWhite } from "/src/api/providers/theme/colors"
```

Comment:
Let's avoid using this colors as variables. You should instead add them in themes.
I'll leave this for now and fix later